grails-app/domain/org/pih/warehouse/product/ProductAssociation.groovy

```diff
@@ -26,7 +26,7 @@ class ProductAssociation {
     String id
     ProductAssociationTypeCode code
     Product associatedProduct
-    BigDecimal quantity = 0
+    BigDecimal quantity = 1
```

Comment:
This feels like a breaking change. What's the reason for this?

Reply:
To get rid of setting conversion by default to `1:0` (or `0:1`) which makes no sense

Reply:
I restored the 0 here, if it might be breaking